addons/account/wizard/account_payment_register.py

```diff
@@ -671,12 +677,20 @@ def _create_payments(self):
                     'to_reconcile': batch_result['lines'],
                     'batch': batch_result,
                 })
-
-        payments = self._init_payments(to_process, edit_mode=edit_mode)
-        self._post_payments(to_process, edit_mode=edit_mode)
-        self._reconcile_payments(to_process, edit_mode=edit_mode)
+        if is_async_payment:
+            payments = self.with_delay()._init_payments_async(to_process, edit_mode=edit_mode)
+        else:
+            payments = self._init_payments(to_process, edit_mode=edit_mode)
+            self._post_payments(to_process, edit_mode=edit_mode)
+            self._reconcile_payments(to_process, edit_mode=edit_mode)
         return payments
 
+    def _post_payments_async(self, to_process, edit_mode=False):
+        return False
```

Comment:
Maybe it will be a good idea to raise a NotImplementedError here. Like that, if the context is present but the methods are not implemented in the project, we raise and error